static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -190,40 +189,40 @@ function useReplayData({replaySlug, orgSlug}: Options): Result {
   );
 
   const fetchErrors = useCallback(
-    async errorIds => {
-      const conditions = new MutableSearch('');
-      errorIds.forEach((errorId, i) => {
-        if (i > 0) {
-          conditions.addOp('OR');
-        }
-        conditions.addFilterValues('id', [errorId]);
-      });
+    async (replayRecord: ReplayRecord) => {
+      if (!replayRecord.errorIds.length) {
+        return [];
+      }
 
       const response = await api.requestPromise(`/organizations/${orgSlug}/events/`, {
         query: {
           field: ['id', 'error.value', 'timestamp', 'error.type', 'issue.id'],
-          projects: [projectSlug],
-          query: conditions.formatString(),
+          projects: [-1],
+          start: replayRecord.startedAt.toISOString(),
+          end: replayRecord.finishedAt.toISOString(),
+          query: `id:[${String(replayRecord.errorIds)}]`,
```

Comment:
This is valid query syntax? 👀

Reply:
yes!
it evaluates to be: `id:[abc, zyx]` which is an OR case.

I tested it against this replay which has 4 errors: https://dev.getsentry.net:7999/organizations/sentry/replays/javascript:22585d5caca7435a860f48b6cd079f0b/